### PR TITLE
API-Reactivity-core translated -review

### PR DESCRIPTION
**Pull request by Navidkhm**
Thanks for reviewing; Whenever notified It'll be edited ASAP.

### CODE REVIEW

File: src/api/reactivity-core.md [hunk 1]
```diff
@@ -21,15 +22,17 @@ Takes an inner value and returns a reactive and mutable ref object, which has a
   }
   ```
 
-- **Details**
+- **جزئیات**
 
-  The ref object is mutable - i.e. you can assign new values to `.value`. It is also reactive - i.e. any read operations to `.value` are tracked, and write operations will trigger associated effects.
+  ‌ref object گفته شده قابل تغییر است - یعنی شما می‌توانید به `.value` مقدار جدید نسبت دهید.
+  همچنین reactive است یعنی هر عملیات خواندن که مربوط به `.value` است رهگیری (tracked) می‌شود.
```
Comment (mostafa-nematpour) [inline src/api/reactivity-core.md:28]:
```suggestion
  ‌ref object گفته شده قابل تغییر است - یعنی شما می‌توانید به `‎.value` مقدار جدید نسبت دهید. همچنین reactive است یعنی هر عملیات خواندن که مربوط به `‎.value` است رهگیری (tracked) می‌شود.
```

File: src/api/reactivity-core.md [hunk 2]
```diff
@@ -21,15 +22,17 @@ Takes an inner value and returns a reactive and mutable ref object, which has a
...
 
-  If an object is assigned as a ref's value, the object is made deeply reactive with [reactive()](#reactive). This also means if the object contains nested refs, they will be deeply unwrapped.
+  اگر یک آبجکت به عنوان یک مقدار از ref در نظر گرفته شود، آن آبجکت با [reactive()](#reactive) عمیقا reactive می‌شود.
+  این جمله همچنین به این معنی است که اگر آن آبجکت ref های تو در تو داشته باشد آنها عمیقا از پوشش خارج می‌شوند (unwrapped).
```
Comment (mostafa-nematpour) [inline src/api/reactivity-core.md:31]:
```suggestion
  اگر یک آبجکت به عنوان یک مقدار از ref در نظر گرفته شود، آن آبجکت با [reactive()‎](#reactive) عمیقا reactive می‌شود. این جمله همچنین به این معنی است که اگر آن آبجکت ref های تو در تو داشته باشد آنها عمیقا از پوشش خارج می‌شوند (unwrapped).
```

File: src/api/reactivity-core.md [hunk 3]
```diff
@@ -39,15 +42,16 @@ Takes an inner value and returns a reactive and mutable ref object, which has a
...
-  - [Guide - Typing `ref()`](/guide/typescript/composition-api#typing-ref) <sup class="vt-badge ts" />
+- **همچنین ببینید**
+  - [راهنما - مبانی Reactivity با `ref()`](/guide/essentials/reactivity-fundamentals#ref)
+  - [راهنما - Typing `ref()`](/guide/typescript/composition-api#typing-ref) <sup class="vt-badge ts" />
 
 ## computed() {#computed}
 
-Takes a getter function and returns a readonly reactive [ref](#ref) object for the returned value from the getter. It can also take an object with `get` and `set` functions to create a writable ref object.
+یک تابع getter می‌گیرد و به ازای مقدار بازگشت داده شده از getter یک آبجکت [ref](#ref) که readonly و reactive است را برمی‌گرداند.
+همچنین می‌تواند یک آبجکت با تابع‌های `get` و `set` برای ایجاد یک آبجکت ref که writable است را بگیرد.
```
Comment (mostafa-nematpour) [inline src/api/reactivity-core.md:52]:
```suggestion
یک تابع getter می‌گیرد و به ازای مقدار بازگشت داده شده از getter یک آبجکت [ref](#ref) که readonly و reactive است را برمی‌گرداند. همچنین می‌تواند یک آبجکت با تابع‌های `get` و `set` برای ایجاد یک آبجکت ref که writable است را بگیرد.
```

File: src/api/reactivity-core.md [hunk 4]
```diff
@@ -186,29 +190,31 @@ Returns a reactive proxy of the object.
...
   ): DeepReadonly<UnwrapNestedRefs<T>>
   ```
 
-- **Details**
+- **جزئیات**
 
-  A readonly proxy is deep: any nested property accessed will be readonly as well. It also has the same ref-unwrapping behavior as `reactive()`, except the unwrapped values will also be made readonly.
+  یک پروکسی readonly عمیق است:
+  هر پراپرتی تو در تویی که دسترسی داشته باشید نیز readonly خواهد بود.
+  علاوه بر این readonly رفتاری مشابه `reactive()` در ref-unwrapping دارد، به استثنای مقادیر unwrapp شده که همچنان readonly ساخته خواهند شد.
```
Comment (mostafa-nematpour) [inline src/api/reactivity-core.md:213]:
```suggestion
  یک پروکسی readonly عمیق است: هر پراپرتی تو در تویی که دسترسی داشته باشید نیز readonly خواهد بود. علاوه بر این readonly رفتاری مشابه `reactive()` در ref-unwrapping دارد، به استثنای مقادیر unwrapp شده که همچنان readonly ساخته خواهند شد.
```

File: src/api/reactivity-core.md [hunk 5]
```diff
@@ -250,17 +256,18 @@ Runs a function immediately while reactively tracking its dependencies and re-ru
...
+  به صورت پیش‌فرض، ناظران درست قبل از رندر شدن کامپوننت اجرا می‌شوند. قرار دادن `flush: 'post'` ناظر را تا بعد از رندر شدن کامپوننت به تعویق می‌اندازد. برای اطلاعات بیشتر [زمانبندی اجرای callback](/guide/essentials/watchers#callback-flush-timing) را ببینید. در موارد نادری، ممکن است به‌کار انداختن (trigger) یک ناظر بلافاصله وقتی که reactive dependency تغییر می‌کند الزامی باشد، مثلا برای باطل‌کردن یک کش. این امکان با استفاده از `flush: 'sync'` میسر است.
+  هر چند که این تنظیم باید با احتیاط استفاده شود، چرا که می‌تواند باعث مشکلاتی در پرفورمنس و data consistency [اگر چندین پراپرتی همزمان آپدیت شوند] شود.
```
Comment (mostafa-nematpour) [inline src/api/reactivity-core.md:266]:
```suggestion
  به صورت پیش‌فرض، ناظران درست قبل از رندر شدن کامپوننت اجرا می‌شوند. قرار دادن `flush: 'post'` ناظر را تا بعد از رندر شدن کامپوننت به تعویق می‌اندازد. برای اطلاعات بیشتر [زمانبندی اجرای callback](/guide/essentials/watchers#callback-flush-timing) را ببینید. در موارد نادری، ممکن است به‌کار انداختن (trigger) یک ناظر بلافاصله وقتی که reactive dependency تغییر می‌کند الزامی باشد، مثلا برای باطل‌کردن یک کش. این امکان با استفاده از `flush: 'sync'` میسر است. هر چند که این تنظیم باید با احتیاط استفاده شود، چرا که می‌تواند باعث مشکلاتی در پرفورمنس و data consistency [اگر چندین پراپرتی همزمان آپدیت شوند] شود.
```

File: src/api/reactivity-core.md [hunk 6]
```diff
@@ -416,15 +423,17 @@ Watches one or more reactive data sources and invokes a callback function when t
...
   ```js
   watch([fooRef, barRef], ([foo, bar], [prevFoo, prevBar]) => {
     /* ... */
   })
   ```
 
-  When using a getter source, the watcher only fires if the getter's return value has changed. If you want the callback to fire even on deep mutations, you need to explicitly force the watcher into deep mode with `{ deep: true }`. Note in deep mode, the new value and the old will be the same object if the callback was triggered by a deep mutation:
+  زمانی که از یک منبع getter استفاده می‌کنیم، ناظر تنها زمانی فعال می‌شود که مقدار برگشت داده شده از getter تغییر کرده باشد .
+  اگر می‌خواهید که callback حتی زمانی که تغییرات عمیق صورت می‌گیرد نیز فعال شود باید صراحتا با حالت عمیق `{ deep: true }`ناظر را مجبور به این کار کنید.
+  لازم به ذکر است که در حالت عمیق، اگر callback با یک تغییر عمیق (deep mutation) به کار افتاده باشد (triggered) مقدار جدید و قدیمی آبجکت‌هایی مشابه خواهند بود:
```
Comment (mostafa-nematpour) [inline src/api/reactivity-core.md:436]:
```suggestion
  زمانی که از یک منبع getter استفاده می‌کنیم، ناظر تنها زمانی فعال می‌شود که مقدار برگشت داده شده از getter تغییر کرده باشد . اگر می‌خواهید که callback حتی زمانی که تغییرات عمیق صورت می‌گیرد نیز فعال شود باید صراحتا با حالت عمیق `{ deep: true }`ناظر را مجبور به این کار کنید. لازم به ذکر است که در حالت عمیق، اگر callback با یک تغییر عمیق (deep mutation) به کار افتاده باشد (triggered) مقدار جدید و قدیمی آبجکت‌هایی مشابه خواهند بود:
```

**Review by mostafa-nematpour — COMMENTED** [review]
Hello,thank your for contribution to the project.
It sounds good
Try to keep the number of lines equal, this helps prevent confusion in receiving updates from the main repo.

Some English words contain symbols, such as method names, poorly displayed.
use this:
Screenshot_2023-12-10_15_06_48

**Comment by Navidkhm** [comment]
Thanks for your review and suggestions.
I added characters and checked I didn't mix the lines wrong;

**Comment by mostafa-nematpour** [comment]
thank you

**Pull request merged**